src/Entity/User.php

```
@@ -0,0 +1,358 @@
+
+    #[ORM\Column(type: Types::DATE_IMMUTABLE)]
+    private \DateTimeImmutable $dateOfBirth;
+
+    #[ORM\Column(type: Types::STRING, length: 255, nullable: true)]
+    private ?string $twoFactorSecret = null;
+
+    /**
+     * @var list<string>|null
+     */
+    #[ORM\Column(type: Types::JSON, nullable: true)]
+    private ?array $twoFactorRecoveryCodes = null;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $twoFactorRecoveryCodes. Keep variable name length under 20. (undefined)

(LongVariable)